lib/server.rb

```diff
@@ -1,45 +1,56 @@
-require 'socket'
 require 'pry'
+require 'socket'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.